staff/lab/ocf-tv

```diff
@@ -62,14 +62,19 @@ def open_vnc(args):
         proc.wait()
 
 
-def tunnel_audio(args=None):
+def tunnel_audio(args):
     port = unused_port()
-    proc = ssh_to_tv(port, 4713)
+    proc = tunnel_to_host(port, 4713, args.host)
 
     try:
         wait_for_port('localhost', port, ssh_proc=proc)
         create_audio_tunnel(port)
-        input('Press enter to close the tunnel...')
+        while True:
+            vol = input('Enter new volume or nothing to close the tunnel: ')
+            if vol:
+                update_volume(check_volume(vol), args.host)
```

Comment:
It would be better UX to not have the script bomb out if the user enters an invalid volume here. Consider catching the ArgumentTypeError and showing some kind of error message.